### ISSUE TITLE
Can't stop propagation of the default events (click)

### ISSUE THREAD

**Issue by Marfusios** [DevExpress/devextreme-angular#583]
Hello, 

thanks for the framework, it's really great. 
I've found out an issue with native events, right now it doesn't work in Angular way. 
I don't know if it is by design, but it complicates situation when we need to provide our custom library on top of DevExtreme (and not to force developers into non-default usage). 

Example: [plunker](http://plnkr.co/edit/854lym5hBLXddlWWMBAh?p=preview)

As you can see, components `MyButton` and `My2Button` are very similar, public API is same, implementation is almost the same. But if we use internally `<dx-button>`, then disabling doesn't work. Event `click` is still propagated. On the other hand with the native button, it works as expected. 

Is it a bug? Would it be too complex to follow Angular style of handling native events?

**Comment by GoshaFighten** [comment]
The `disabled` option of our button is in effect only if the `onClick` event of the button is raised or not (and of cause, the button style). If you handle the native `click` event, this event won't be prevented. I suggest you use the `HostBinding` directive inside your `my2-button` component to change the `pointer-events` CSS style of the component host:
 
``` TypeScript
@Input()
get disabled(): boolean { return this._disabled; }
set disabled(value: boolean){ this._disabled = value; this.events = this._disabled ? "none" : "auto"; }
public _disabled: boolean = false;
@HostBinding('style.pointer-events') events = "auto"; 
```
 
See the updated [plunk](http://plnkr.co/edit/6wGUO1P2nkuDyjGFITXq?p=preview). Will this work for you?

**Comment by Marfusios** [comment]
As we discussed with your college on [support center](https://www.devexpress.com/Support/Center/Question/Details/T566700/dxbutton-cannot-stop-propagation-of-the-default-events-click), right now I solved it by handling `click` event also on container div, see [plunker](http://plnkr.co/edit/CPTcLTdJwKGaBNcUXCO6?p=preview). 

But your solution also seems feasible, thanks. I will try both and let you know in case of any long-time issue.
